src/Eshopworld.Messaging/QueueAdapter.cs

```diff
@@ -62,7 +62,7 @@ internal async Task Send([NotNull]T message)
             var qMessage = new Message(Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(message)))
             {
                 ContentType = "application/json",
-                Label = message.GetType().FullName
+                Label = message.GetType().Name
```

Comment:
**This is a no go** (and all changes like this), because it creates a reasonable enough clash surface in a scenario before the Service Buses aren't tenanted, i.e. a payments message `Payments.Events.OrderCompleted` will go to the same transport channel as `Checkout.Events.OrderCompleted` on different payloads and different routing schemes.

Reply:
Sure makes sense. I'll drop this.